misc/python/materialize/cli/mzconduct.py

```diff
@@ -795,30 +813,48 @@ class ChaosDockerWorkflowStep(WorkflowStep):
     def __init__(
         self,
         service: str,
-        loop: bool,
+        other_service: str,
+        duration: int,
```

Comment:
~should either of these have a default `None` or something?~

Oh I see this is just a base class. Since neither of the inheriting classes do anything with this should the defaults be specified here and you could just delete the `__init__` methods in the subclasses?

Reply:
Moving the defaults to the base class! The inheriting classes are still passing the docker commands (`stop` v `pause`), so I will have to keep the `__init__` for now.

Reply:
Actually, I need the defaults on the inheriting classes because I want some of the inputs to be optional.